tests/ui/imports/auxiliary/simple-dylib.rs

```diff
@@ -0,0 +1,4 @@
+//@ compile-flags --crate-type=dylib -Cprefer-dynamic
```

Comment:
Question: I *think* this needs to be

```
//@ compile-flags: --crate-type=dylib -Cprefer-dynamic
```

but then again I don't recall exactly what compiletest directive parsing accepts. The `--crate-type=dylib` is redundant though because of the in-source `crate_type` attribute.

Reply:
cc https://github.com/rust-lang/rust/issues/123760